github/util.py

```diff
@@ -63,15 +65,14 @@ def _create_repository(
 
     def create_or_update_file(
         self,
-        repository_branch: str,
```

Comment:
make this an optional parameter (default value =None - if None, use default_branch)